### ISSUE TITLE
EHealth.Web.DeclarationsController#index error

### ISSUE THREAD

**Issue by lymychp** [edenlabllc/ehealth.api#2101]
error from logs
```json
    "phoenix": {
      "request_id": "9b5bb1f1-7bab-4c8c-9729-9be3209b2896#23011",
      "message": "No function clause matching in EHealth.Web.FallbackController.call/2: {:error, \"\"}",
      "log_type": "error"
    },
```

**Comment by lymychp** [comment]
https://github.com/ehealth-ua/api-spec/issues/300

**Comment by lymychp** [comment]
@PavelVesnin still reproducible
```
  "phoenix": {
      "status": 500,
      "request_id": "cf9b5399-1a6c-4140-9201-9cbe5fd567e0#22514",
      "path": "/api/declarations",
      "method": "GET",
      "log_type": "http",
      "handler": "Elixir.EHealth.Web.DeclarationsController#index",
      "duration": 145.072,
      "date_time": "2018-04-24T04:37:57Z",
      "api_version": "N/A"
```

**Comment by PavelVesnin** [comment]
@fixed and builded container [7.50.6](https://hub.docker.com/r/edenlabllc/ehealth/tags/)

**Comment by lymychp** [comment]
delivered to prod
